pseudopotential_calculator/calculations/slab.py

```diff
@@ -101,80 +162,23 @@ def get_surface(
     n_fixed_layer: int = 0,
     n_free_layer: int = 0,
     n_vacuum_layer: int = 0,
+    adsorbate_height: float = 0,
 ) -> Atoms:
     n_layer = n_free_layer + n_fixed_layer
     slab = cast(Atoms, surface(atoms, slab_direction, n_layer, 0))
 
     height_per_layer = _get_height_per_layer(atoms, slab_direction)
-    add_vacuum(slab, n_vacuum_layer * height_per_layer)
+    add_vacuum(
```

Comment:
doesn't actually add the adsorbate - why is that extra argument necessary

Reply:
this just takes the adsorbate into account and add more vacuum on the slab automatically